designs/administrator/olm/list-of-operators/list-of-operators.md

```diff
@@ -0,0 +1,57 @@
+
+  b. “All Namespaces” is a text only link that is associated with the Namespaces that fall across all Namespaces.
+
+  c. Thirdly, there is a multi-project view. For example, the Portworx Operator falls across multiple projects but not all projects, so in this example the Managed Namespaces column will have a popover on click. The Namespace names should be listed underneath as seen in the mockup.
+
+![3scale-all-projects-view1](img/3scale-all-projects-view1.jpg)
+On the Managed Namespaces column, when the user clicks the the popover it should read “Operands in these Namespace(s) are managed by the Operator”
+
+![3scale-all-projects-view2](img/3scale-all-projects-view2.jpg)
+On this screen the Portworx Operator falls across multiple projects but not all projects, so in this example the Managed Namespace column will have a popover on click.
+
+![3scale-all-projects-view3](img/3scale-all-projects-view3.jpg)
+Clicking into the 2 Namespaces field, the popover will appear. The Managed Namespace names should be listed underneath as seen in the mockup.
+
+![3scale-all-projects-view](img/3scale-all-projects-view.jpg)
+On the all-projects view when the user clicks into the 3scale Operator they will be taken to the Operator Details page.
+
+![all-projects-operator-details](img/all-projects-operator-details.jpg)
+If the user clicked on an Operator in the list view while the project selector is 'all-projects,' the details view would have the project selector = to that Operator's Namespace since we wouldn't have the context of a single Managed Namespace to use. The Namespace field will read Managed Namespace. Zooming in on the interaction of the popover on the CSV Overview section on the Operator Details page:
+![all-projects-operator-details1](img/all-projects-operator-details1.jpg)
+
+## Single Project view
+
+![tooltip-my-project-managed-namespace](img/tooltip-my-project-managed-namespace.jpg)
+On the single project view, the Namespace column does not appear here because the user who has access to “single project” view doesn’t necessarily have the access to Namespace but will have access to the “Managed Namespaces” column. This is how this view will look on a single-project view:
+
+![3scale-single-project-view](img/3scale-single-project-view.jpg)
+Clicking into 3scale Operator, on the my-project Installed Operators page, if the user clicked on an Operator in the list view while the project selector is a single Namespace, the details view would also be that same single Namespace.
+
+![all-projects-operator-details](img/all-projects-operator-details.jpg)
+Here is the in-line Managed Namespace field under the CSV information. The ClusterServiceVersion Overview will always show the "Managed Namespace".
+
+## Select Operator with All Namespaces under Managed Namespaces
+
+![Jaeger-my-project-view](img/Jaeger-my-project-view.jpg)
+User selects Jaeger Operator
+
+![Jaeger-details](img/Jaeger-details.jpg)
```

Comment:
I _think_ this operator details screen would show "All Namespaces" in the **Managed Namespaces** field since that is what is shown in the list view (and I believe they would match?) It may also be worth showing that the **Namespace** field also still exists here (maybe covered by the popover in this image?) @tlwu2013

Reply:
+1...this example is All Namespaces, so should we again list the namespaces here?

Reply:
A technical note about the popover here: today all of the popover text that appears when clicking a field heading with a dotted underline comes from [Kubernetes' OpenAPI Specification](https://github.com/kubernetes/kubernetes/tree/master/api/openapi-spec) (see the descriptions in the enormous `swagger.json` file there). If you go to the in-Console YAML editor and hover your mouse over various fields like `name` or `annotations`, the same descriptions appear there as well.

`Managed Namespace` isn't currently described in that OpenAPI thingie today, so I think this mockup would be the first time we've co-opted that popover to show custom description text, which could be tricky.

I notice that we don't have dotted underline descriptions for other fields like `Operator Deployments`, `Operator Service Accounts`, or `Operator Group` either. @dmesser @tlwu2013 is there a way we could add these descriptions in a programatic way similar to (or using) OpenAPI?

Reply:
I think the `Managed Namespace` is the first time we are co-opting the popover. It was discussed with Sam/Ali/Tony and others and we think it is justified and seemingly doable. I think part of the justification of retaining `Namespace` and just adding `Managed Namespace` instead of replacing was to retain the OpenAPI info about `Namespace` (among other reasons.)

Reply:
@rachael-phillips - Another place to note the use of popover if we end up heading down that path!